pkgs/servers/web-apps/discourse/update.py

```diff
@@ -187,9 +187,13 @@ def update_plugins():
 
     """
     plugins = [
+        {'name': 'discourse-calendar'},
         {'name': 'discourse-canned-replies'},
+        {'name': 'discourse-checklist'},
+        {'name': 'discourse-data-explorer'},
         {'name': 'discourse-github'},
         {'name': 'discourse-math'},
+        {'name': 'discourse-migratepassword'},
```

Comment:
this is from a different org, should be `{'name': 'discourse-migratepassword', 'owner': 'discoursehosting'},`

Reply:
Should be resolved in latest force push.